Makefile

```diff
@@ -144,9 +146,13 @@ manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and Cust
 generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
+GOLANGCI_TMP_FILE = .golangci.mktmp.yml
 .PHONY: fmt
-fmt: ## Run go fmt against code.
+fmt: golangci-lint yq ## Run go fmt against code.
 	go fmt ./...
+	$(YQ) e '.linters = {"disable-all": true, "enable": ["gci"]}' .golangci.yml  > $(GOLANGCI_TMP_FILE)
+	$(GOLANGCI_LINT) run --config=$(GOLANGCI_TMP_FILE) --fix
```

Comment:
I'm wondering would it be ok to remove --fix from the lint target. To have clear check without surprises in changing the source. I personally would prefer that (may time run make lint on uncommitted changes and that manually splitted the changes). But it's up to others what is more convenient.

Reply:
I would be fine with that, but as a lazy dev, I would love to have it easily fixable too. Maybe by introducing a second target called `lint-autofix` or something.